spec/json/BrandedFaresUpsell_v1_swagger_specification.json

```diff
@@ -1,7 +1,7 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "1.0.1",
+    "version": "1.2.0",
```

Comment:
why its 1.2.0 and not 1.1.0?

Reply:
Because we missed an update that was Enterprise only. I choose to keep the alignment, it's easier to follow the loads